README.md

```diff
@@ -1,58 +1,53 @@
-* [RHSCL Container CI](https://github.com/sclorg/rhscl-container-ci): Scripts that help testing containers on RHEL platform.
+## Repository Overview
 
+- [Container Contribution Guidelines](contribution.md): Learn about what principles to follow when contributing to the sclorg organization, particularly in relation to container repositories.
+- [Container Images Overview Based on Software Collections](images.md): Access Dockerfiles, relevant sources, and tests for container images.
 
-Where to Learn More About Containers
-====================================
-There is a lot of articles around Docker and containers generally, these are just few that seemed to provide basic introduction:
+- [scl-utils](https://github.com/sclorg/scl-utils): The upstream project for the scl-utils package available in RHEL, CentOS, and Fedora, enabling the building of RPM packages in an alternative directory.
+- [spec2scl](https://github.com/sclorg/spec2scl): A Python-based tool that facilitates the migration of RPM spec files into SCL-enabled variants by adding macros, scriptlets, and more.
+- [softwarecollections.org](https://github.com/sclorg/softwarecollections): Source code for the [softwarecollections.org](http://softwarecollections.org/) website, a Django application that serves as a directory for available Software Collections.
+- [centpkg for SCLo](https://github.com/sclorg/centpkg-sclo): A compact wrapper that assists [SCLo SIG](http://wiki.centos.org/SpecialInterestGroup/SCLo) maintainers in building SCL packages in [CBS](http://cbs.centos.org/).
+- [centos-release package](https://github.com/sclorg/centos-release-scl): A minimal RPM package with a YUM repository, offering upstream Software Collections built in [CBS](http://cbs.centos.org/) by the community (excluding those in RHSCL).
+- [SCLo CI Tests](https://github.com/sclorg/sclo-ci-tests): Tests executed in [CentOS CI](http://ci.centos.org) to verify the successful (re)building of RPM packages in [CBS](http://cbs.centos.org/).
+- [RPM List Builder](https://github.com/sclorg/rpm-list-builder): A tool designed to facilitate the rebuilding of SRPMs in [CBS](http://cbs.centos.org/) that were previously available in different repositories (e.g., RHSCL and others).
+- [RHSCL Container CI](https://github.com/sclorg/rhscl-container-ci): Scripts for testing containers on the RHEL platform.
 
- * https://developer.fedoraproject.org/tools/docker/about.html
- * https://fedoramagazine.org/quick-containers-with-fedora-dockerfiles/
- * https://www.liquidweb.com/kb/how-to-list-and-attach-to-docker-containers/
- * https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/overview_of_containers_in_red_hat_systems/introduction_to_linux_containers
+# Where to Learn More About Containers
```

Comment:
In my opinion is redundant to state such a generic information in SCLOrg's readme.